client.js

```diff
@@ -1,41 +1,78 @@
-const WebSocketClient = require("ws");
+const WebSocket = require("ws");
 require("dotenv").config();
```

Comment:
Nitpick: feels like this line should be below all the other imports because it's actually invoking a function.

Reply:
Alternatively, change this to:
```suggestion
const dotenv = require("dotenv");
```

And then call `dotenv.config()` below the other imports.

Reply:
Good point. Moved the line below imports.
Do you think I should do the second option for aesthetics?

Reply:
No, we can leave it as `require("dotenv").config()` as the only reason to even require the package is to immediately call `config` on it.

Normally though, yes, you would use the `const foo = require("bar")` style.